BEIMA.Client/cypress/integration/DevicePage.spec.js

```diff
@@ -25,23 +25,23 @@ describe('Device Page', () => {
     cy.get('[id=deviceLatitude]').should('exist')
     cy.get('[id=deviceLongitude]').should('exist')
     cy.get('[id=locationNotes]').should('exist')
-    cy.get('[id=deviceTag]').should('exist')
-    cy.get('[id=deviceModelNumber]').should('exist')
-    cy.get('[id=deviceSerialNumber]').should('exist')
-    cy.get('[id=deviceManufacturer]').should('exist')
-    cy.get('[id=deviceYearManufactured]').should('exist')
+    cy.get('[id=fields]').within(() => {
+      cy.get('input').each((val, index, collection) => {
+        cy.wrap(val).should('exist')
```

Comment:
Doing it this way grabs the inputs on the page, and checks if it's there, dynamically. We aren't actually checking if the correct input field is there. I think how we had it before was good. We had actual expected results of what should be on the page

Reply:
Good point.  If those fields and custom fields don't get returned for some reason this test would still pass.